content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -377,6 +377,8 @@ when satisfying the Pod (anti)affinity.
 The keys are used to look up values from the pod labels; those key-value labels are combined
 (using `AND`) with the match restrictions defined using the `labelSelector` field. The combined
 filtering selects the set of existing pods that will be taken into Pod (anti)affinity calculation.
+It's not recommended to use `matchLabelKeys` with labels that might be updated 
+because the update of the label isn't reflected onto the merged `LabelSelector`.
 
```

Comment:
```suggestion
filtering selects the set of existing pods that will be taken into Pod (anti)affinity calculation.

{{< warning >}}
It's not recommended to use `matchLabelKeys` with labels that might be updated 
because the update of the label isn't reflected onto the merged `LabelSelector`.
{{< /warning >}}

```